sklearn/metrics/tests/test_classification.py

```diff
@@ -1374,6 +1374,29 @@ def test_log_loss():
     loss = log_loss(y_true, y_pred)
     assert_almost_equal(loss, 1.0383217, decimal=6)
 
+    #test labels option
+    X = [[1,1], [1,1], [2, 2], [2, 2]]
+
+    y = [1, 1, 2, 2]
+    clf = DecisionTreeClassifier()
+    clf.fit(X, y)
+
+
+    y_score = clf.predict_proba([[2,2], [2,2]])
```

Comment:
As said before, something in between 0 and 1, might be better, to avoid the clipping done below.